core/embed/rust/build.rs

```diff
@@ -77,24 +84,28 @@ fn prepare_bindings() -> bindgen::Builder {
         "-I../../vendor/micropython/lib/uzlib",
         "-I../lib",
         "-I../trezorhal",
-        "-DSTM32F427xx",
+        format!("-D{}", mcu_type()).as_str(),
         format!("-DTREZOR_MODEL_{}", model()).as_str(),
         format!("-DTREZOR_BOARD=\"{}\"", board()).as_str(),
     ]);
 
     // Pass in correct include paths and defines.
     if is_firmware() {
-        bindings = bindings.clang_args(&[
-            "-nostdinc",
-            "-I../firmware",
-            "-I../trezorhal/stm32f4",
-            "-I../../build/firmware",
-            "-I../../vendor/micropython/lib/stm32lib/STM32F4xx_HAL_Driver/Inc",
-            "-I../../vendor/micropython/lib/stm32lib/CMSIS/STM32F4xx/Include",
-            "-I../../vendor/micropython/lib/cmsis/inc",
-            "-DUSE_HAL_DRIVER",
-            "-DSTM32_HAL_H=<stm32f4xx.h>",
-        ]);
+        let mut clang_args: Vec<&str> = Vec::new();
+
+        let includes = env::var("RUST_INCLUDES").unwrap();
```

Comment:
This will probably fail when running some `cargo ...` commands directly. I don't have better solution though, we can leave it for now and solve it when needed.